src/pages/Buy/Buy.tsx

```diff
@@ -48,7 +47,8 @@ export const Buy = () => {
   useGetFiatRampsQuery()
 
   const { chainId, assetSubId } = useParams<MatchParams>()
-  const [selectedAssetId, setSelectedAssetId] = useState<AssetId>(ethAssetId)
+  const [selectedAssetId, setSelectedAssetId] = useState<AssetId>()
```

Comment:
Ohh interesting, the generic is automagically `T | undefined` meaning we don't have to explicitly state it, nice find! My brain broke for a second on how this worked.

<img width="839" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f4fe457e-9d9f-4faf-8803-f114c7b4412a">

May be worth keeping the undefined union for this precise consistency reason however, though if we all agree that's cleaner, we may just ditch explicit undefined unions for `useState` generic everywhere in the app